### PR TITLE
[Doc] Add guides for end-to-end encryption in Node.js client

### PR DESCRIPTION
**Pull request by Anonymitaet**
Add docs for https://github.com/apache/pulsar-client-node/pull/138.

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@zymap @tuteng @Huanli-Meng @Jennifer88huang could you please help review? Thanks

**Review by Jennifer88huang — APPROVED** [review]

**Review by zymap — APPROVED** [review]

File: site2/docs/client-libraries-node.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Compatibility between each version of the Node.js client and the C++ client is a
 | 1.0.0          | 2.3.0 or later |
 | 1.1.0          | 2.4.0 or later |
 | 1.2.0          | 2.5.0 or later |
-| 1.3.0          | 2.7.0 or later |
+| 1.3.0          | 2.7.1 or later |
```
Comment (tuteng) [inline site2/docs/client-libraries-node.md:30]: I think 1.3.0 does not support this feature, we need to wait for the next release of the node client

**Review by tuteng — CHANGES_REQUESTED** [review]

File: site2/docs/client-libraries-node.md (hunk 1 above)
Comment (Anonymitaet) [reply site2/docs/client-libraries-node.md]: @tuteng OK, then I've removed `| 1.3.0          | 2.7.1 or later |` for both master and 2.7.2. PTAL.

**Review by tuteng — APPROVED** [review]

**Comment by eolivelli** [comment]
I am removing 2.7.2 label,
does are built from master, correct ?
@Anonymitaet

**Comment by Anonymitaet** [comment]
@eolivelli I've added docs for both 2.7.2 and master.

**Pull request merged**